src/static/chemiscope.css

```diff
@@ -205,9 +208,61 @@
     font-weight: bold;
 }
 
+/* Plotly Modebar */
+
+.modebar-group {
+    padding-left: 0 !important;
+}
+
+.modebar-btn {
+    cursor: default !important;
+}
+
+.modebar-btn::after {
+    opacity: 0.9 !important;
+    top: 140% !important;
+}
+
+.icon:hover + .modebar-btn::after {
+    visibility: visible;
+}
+
+.modebar-btn::before {
+    content: none !important;
+    font-family: Arial !important;
```

Comment:
Why change the font family? Are we also changing it for the other tooltips?

Reply:
I am changing it to better match with the fonts of the structure.